.travis.yml

```diff
@@ -70,8 +72,30 @@ matrix:
       python: "3.6"
       env: TOXENV=py36-flake8
 
+    - stage: Deploy
+      name: Publishing current Git tagged version of dist to PyPI
+      if: tag IS present
```

Comment:
In general, you might want to push without `v` part or with some a/b/dev/rc suffices, so it would require a way more complex regexp. I think since there's only a few people doing releases it's an unneeded complication — all of those probably know which tags to use. Also, setuptools-scm itself validates tags against PEP 440.